packages/components/header/_header.scss

```diff
@@ -26,155 +26,81 @@
   @include govuk-width-container;
   padding: 20px 0;
 
-  @include mq($until: tablet) {
-    padding-bottom: 0;
-  }
-
   @include mq($from: tablet) {
-    border-bottom: 1px solid $color_transparent_nhsuk-white-20;
     display: flex;
+    gap: nhsuk-spacing(3);
     justify-content: space-between;
   }
 }
 
 .nhsuk-header__logo {
-  @include mq($until: tablet) {
-    position: relative;
-    z-index: 1;
-  }
-
-  .nhsuk-logo__background {
-    fill: $color_nhsuk-white;
-
-    @include mq($media-type: print) {
-      fill: $color_nhsuk-blue;
-    }
-  }
-
-  .nhsuk-logo__text {
-    fill: $color_nhsuk-blue;
-
-    @include mq($media-type: print) {
-      fill: $color_nhsuk-white;
-    }
-  }
+  align-items: center;
+  display: flex;
+  gap: nhsuk-spacing(2) nhsuk-spacing(3);
 
-  @include mq($from: tablet) {
-    padding-left: 0;
+  @include mq($until: mobile) {
+    align-items: start;
+    flex-direction: column;
   }
+}
 
-  .nhsuk-logo {
-    @include nhsuk-logo-size;
-    border: 0;
-  }
+.nhsuk-header__link,
+.nhsuk-header__service-link {
+  @include nhsuk-link-style-white;
+  text-decoration: none;
 
-  @include mq($until: desktop) {
-    max-width: 60%;
+  &:hover {
+    text-decoration: underline;
   }
 
-  @media (max-width: 450px) {
-    max-width: 50%;
+  &:active {
+    color: $color_nhsuk-white;
   }
 }
 
 .nhsuk-header__link {
-  @include nhsuk-logo-size;
   display: block;
 
   &:hover {
-    .nhsuk-logo {
-      box-shadow: 0 0 0 $nhsuk-focus-width $color_shade_nhsuk-blue-35;
-    }
+    box-shadow: 0 0 0 $nhsuk-focus-width $color_shade_nhsuk-blue-35;
   }
 
   &:focus {
-    box-shadow: none;
-
-    .nhsuk-logo {
-      box-shadow:
-        0 0 0 $nhsuk-focus-width $nhsuk-focus-color,
-        0 $nhsuk-focus-width 0 $nhsuk-focus-width $nhsuk-focus-text-color;
-    }
-  }
-
-  @include mq($media-type: print) {
-    &:after {
-      content: "";
-    }
+    box-shadow: 0 0 0 $nhsuk-focus-width $nhsuk-focus-color,
+      0 $nhsuk-focus-width 0 $nhsuk-focus-width $nhsuk-focus-text-color;
   }
 
-  &:hover,
-  &:active,
-  &:focus {
-    background-color: transparent;
+  .nhsuk-logo {
+    @include nhsuk-logo-size;
+    display: block;
   }
 }
 
-.nhsuk-header__logo--only {
-  max-width: 100%;
-
-  @include mq($until: tablet) {
-    padding-bottom: nhsuk-spacing(3);
-  }
-  @include mq($from: tablet) {
-    .nhsuk-header__link--service {
-      align-items: center;
-      display: flex;
-      -ms-flex-align: center;
-      margin-bottom: 0;
-      width: auto;
-    }
-
-    .nhsuk-header__service-name {
-      padding-left: nhsuk-spacing(3);
-    }
-  }
-}
-
-.nhsuk-header__content {
-  @include print-hide();
-  position: relative;
-
-  &.js-show {
-    border-bottom: nhsuk-spacing(1) solid $color_nhsuk-grey-5;
-  }
-
-  @include mq($from: tablet) {
-    &.js-show {
-      border-bottom: 0;
-    }
-  }
+.nhsuk-header__service-link {
+  @include nhsuk-font(19, $line-height: 22px);
+  text-wrap: balance;
```

Comment:
Can I suggest something a bit ugly? 

If we add 
`margin-top: -2px;
  margin-bottom: -3px;`
to this class.

Then the logo won't be pulled down slightly when the service name wraps. (it's only a couple pixels so perhaps not worth this ugly code?)